webapp/.gitignore

```diff
@@ -1,2 +1,3 @@
+.eslintcache
+junit.xml
 node_modules
-.npminstall
```

Comment:
It was an old relic of our web app Makefile. We used to have a Makefile target that installed dependencies and then created an `.npminstall` file to mark that that had been done instead of just making the Makefile rule targeting the `node_modules` folder itself.

We should just remove the file instead of committing it now

Reply:
Dropped the file